rpcs3/rpcs3qt/pad_led_settings_dialog.cpp

```diff
@@ -0,0 +1,98 @@
+﻿#include "pad_led_settings_dialog.h"
+#include <QColorDialog>
+
+pad_led_settings_dialog::pad_led_settings_dialog(QDialog* parent)
+    : QDialog(parent), ui(new Ui::pad_led_settings_dialog)
+{
+	ui->setupUi(this);
+	setFixedSize(size());
+
+	connect(ui->b_save, SIGNAL(released()), this, SLOT(close_dialog()));
```

Comment:
We don't use this connect style
```suggestion
	connect(ui->b_save, &QAbstractButton::clicked, this, &pad_led_settings_dialog::accept);
```